egnyte/resources.py

```diff
@@ -284,7 +284,7 @@ def update(self, email=None, familyName=None, givenName=None, active=None, sendI
         """
         url = self._client.get_url(self._url_template, id=self.id)
         name = base.filter_none_values(dict(familyName=familyName, givenName=givenName)) or None
-        data = base.filter_none_values(dict(email=email, active=active, name=name, sendInvite=sendInvite, authType=authType, idpUserId=idpUserId, userPrincipalName=userPrincipalName))
+        data = base.filter_none_values(dict(email=email, active=active, name=name, sendInvite=sendInvite, authType=authType, userType=userType, idpUserId=idpUserId, userPrincipalName=userPrincipalName))
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length - please, make this line to be 79 characters-long as pep8 requires.